engine/src/object.h

```diff
@@ -450,7 +450,8 @@ struct MCPatternInfo
 	MCPatternRef pattern;
 };
 
-class MCObject : public MCDLlist
+class MCObject :
+  public MCDLlist
 {
```

Comment:
Some extraneous whitespace changes here, probably not worth reverting but it proves I was looking properly, right ;-)